<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>amber</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="clipboardCopy">
      <property name="text">
       <string>auto clipboard copy</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="pageHome">
       <layout class="QGridLayout" name="gridLayout_52">
        <item row="1" column="0">
         <widget class="QPushButton" name="homeWikiButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <pointsize>48</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>GITHUB WIKI</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="homeReportButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <pointsize>48</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>REPORT AN ISSUE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="homeRepoButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <pointsize>48</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>GITHUB REPOSITORY</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="homeReleasesButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <pointsize>48</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RELEASES</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageStutter">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="Stutter" name="widgetStutter" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageCopier">
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="Copier" name="widgetCopier" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageTpb">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="TwoPointBezier" name="widgetTpb" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNorm">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="Normalizer" name="widgetNorm" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAlter">
       <layout class="QGridLayout" name="gridLayout_44">
        <item row="1" column="0">
         <widget class="Alter" name="widgetAlter" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionStutter"/>
   <addaction name="actionCopier"/>
   <addaction name="actionTPB"/>
   <addaction name="actionNormalizer"/>
   <addaction name="actionAlter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStutter">
   <property name="text">
    <string>Stutter</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionTPB">
   <property name="text">
    <string>TPB</string>
   </property>
  </action>
  <action name="actionNormalizer">
   <property name="text">
    <string>Normalizer</string>
   </property>
  </action>
  <action name="actionAlter">
   <property name="text">
    <string>Alter</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Alter</class>
   <extends>QWidget</extends>
   <header>include/function/alter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Normalizer</class>
   <extends>QWidget</extends>
   <header>include/function/normalizer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Copier</class>
   <extends>QWidget</extends>
   <header>include/function/copier.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Stutter</class>
   <extends>QWidget</extends>
   <header>include/function/stutter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TwoPointBezier</class>
   <extends>QWidget</extends>
   <header>include/function/twopointbezier.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
